usagov/usagov-2021
# Usagov 1919 consistent alert messages update

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1919

## Description

Make alert components accessible with a screen reader
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Clear you cache (bin/drush cr)
2. Goto http://localhost/avoid-foreclosure
3. Start Voice Over on Mac (command F5) and tab through the left navigation
4. Before you get the the alert component you need to use the arrow key to get to the component, hold down the "control" and "option" keys together while using the right arrow key.
5. You should hear the screen reader announce "info important, region".
6. Continue to hold down the "control" and "option" keys together while using the right arrow key and you should hear the word "info" and then the heading and body text of the alert component.

<img width="1433" alt="Screenshot 2024-10-22 at 2 09 51 PM" src="https://github.com/user-attachments/assets/cfbdf38a-a1f5-4965-be14-d7737e507f85">


### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
